pennylane/operation.py

```diff
@@ -705,7 +705,7 @@ def compute_matrix(*params, **hyperparams):  # pylint:disable=unused-argument
         The canonical matrix is the textbook matrix representation that does not consider wires.
         Implicitly, this assumes that the wires of the operator correspond to the global wire order.
 
-        .. seealso:: :meth:`.Operator.matrix` and :func:`qml.matrix`
+        .. seealso:: :meth:`.Operator.matrix` and :func:`~pennylane.matrix`
```

Comment:
think this was me. because they're both called "matrix", I want it to be clear what each is referring to. I believe this proposed change will alias accordingly (I stole it from elsewhere in the code), which is sadly needed because people import as qml.
```suggestion
        .. seealso:: :meth:`.Operator.matrix` and :func:`qml.matrix() <pennylane.matrix>`
```